### ISSUE TITLE
Series with name 'time' is shown as No Data in most panels 

### ISSUE THREAD

**Issue by nikut** [grafana/grafana#18479]
**What happened**:

In Grafana 6.3.2 a series aliased to 'time' does not show up in graphs at all. Same happens if the series is aliased 'date' (or I assume some other reserved words). In 5.3.2 this works fine. Changing the alias even slightly (e.g. adding a space at the end) makes it show up again.

**What you expected to happen**:

The series should show in graphs even if its alias is 'time' or 'date'. 

**How to reproduce it (as minimally and precisely as possible)**:

Add a series from a Graphite data source as follows:
```
alias(foo.bar.time, 'time') or
aliasByNode(foo.bar.time, 2)
```
Observe the series doesn't show in the graph. Add it like this instead (note the extra space) and it works:
```
alias(foo.bar.time, 'time ')
```

**Anything else we need to know?**:

**Environment**:
- Grafana version: 6.3.2
- Data source type & version: Graphite (graphite-api 1.1.3)
- OS Grafana is installed on: Official Docker image on Amazon Linux
- User OS & Browser: Mac OS & Chrome
- Grafana plugins:
- Others:

**Comment by torkelo** [comment]
Sounds like an issue with Graphite? Can you show what your query returns in the Query Inspector (in panel edit mode click Query Inspector)

**Comment by nikut** [comment]
According to the Query Inspector Graphite returns the series just fine:
```
response:Array[3]
  0:Object
    target:"time"
    datapoints:Array[288]
  1:Object
    target:"failure"
    datapoints:Array[288]
  2:Object
    target:"time 95th percentile"
    datapoints:Array[288]
```
Also, this works with the same Graphite backend on Grafana 5.3.2 so looks like a Grafana issue to me.

I'm guessing there's some sanitizing going on that breaks the use of Javascript reserved words as series names/aliases?

**Comment by torkelo** [comment]
@ryantxu must be something in the legacy to DataFrame conversion?

**Comment by torkelo** [comment]
Oh, I can replicate the issue you reported we will fix this soon in a patch release.

**Comment by ryantxu** [comment]
With our conversion to DataFrame, we assume that columns named "time" are time values, and most panels are configured to show all non-time data.

I'll look at some options to improve this.

**Comment by ryantxu** [comment]
This is fixed after we apply #18391 -- essentially the `toDataFrame( SeriesData )` forces first column type as number and second as time.  

I'll make a similar PR that could apply to 6.3
